Draggable.js

```diff
@@ -22,28 +22,28 @@ function clamp(number, min, max) {
 
 export default function Draggable(props) {
   const {
-    renderText,
+    renderText = '+',
     isCircle,
-    renderSize,
+    renderSize = 36,
     imageSource,
     renderColor,
     children,
-    shouldReverse,
+    shouldReverse = false,
     onReverse,
-    disabled,
-    debug,
+    disabled = false,
+    debug = false,
     animatedViewProps,
     touchableOpacityProps,
-    onDrag,
-    onShortPressRelease,
-    onDragRelease,
-    onLongPress,
-    onPressIn,
-    onPressOut,
-    onRelease,
-    x,
-    y,
-    z,
+    onDrag = () => {},
+    onShortPressRelease = () => {},
+    onDragRelease = () => {},
+    onLongPress = () => {},
+    onPressIn = () => {},
+    onPressOut = () => {},
+    onRelease = () => {},
+    x = 0,
+    y = 0,
+    z = 1,
```

Comment:
We want to stick with defaultProps for js consumers so that they can get intellisense recommendations without using our types files. Switching it to use default values won't show them intellisense recommendations